[build] `make install` doesn't work on RPI3

### Issue description

```make PLATFORM=PLATFORM_RPI ```

result:

```
ar rcs ../src/libraylib.a rcore.o rshapes.o rtextures.o rtext.o utils.o rmodels.o raudio.o
raylib static library generated (libraylib.a) in ../src!
```

```sudo make install```

result:

```
mkdir --parents --verbose /usr/local/lib
mkdir --parents --verbose /usr/local/include
cp --update --verbose /home/pi/src/raylib/libraylib.a /usr/local/lib/libraylib.a
cp: impossibile eseguire stat di '/home/pi/src/raylib/libraylib.a': File o directory non esistente
make: *** [Makefile:629: install] Error 1
```

It tries to copy ```/home/pi/src/raylib/libraylib.a``` instead of ```/home/pi/src/raylib/src/libraylib.a```

### Environment

rpi3

RPI 2 Model B
OS: RapiOS (Buster)
GL Driver: FULL_KMS

I followed the wiki instructions:
cd src && make PLATFORM = PLATFORM_RPI
and i have same errors:
```
sudo make install
mkdir --parents --verbose /usr/local/lib
mkdir --parents --verbose /usr/local/include
cp --update --verbose /home/pi/raylib/libraylib.a /usr/local/lib/libraylib.a
cp: cannot stat '/home/pi/raylib/libraylib.a': No such file or directory
make: *** [Makefile:635: install] Error 1
```

--- a/src/Makefile
+++ b/src/Makefile
@@ -147,7 +147,7 @@ endif
 # RAYLIB_SRC_PATH adjustment for different platforms.
 # If using GNU make, we can get the full path to the top of the tree. Windows? BSD?
 # Required for ldconfig or other tools that do not perform path expansion.
-ifeq ($(PLATFORM),PLATFORM_DESKTOP)
+ifeq ($(PLATFORM), $(filter $(PLATFORM),PLATFORM_DESKTOP PLATFORM_RPI))
     ifeq ($(PLATFORM_OS),LINUX)
         RAYLIB_PREFIX  ?= ..
         RAYLIB_SRC_PATH = $(realpath $(RAYLIB_PREFIX))

